test/run_tests.py

```diff
@@ -85,79 +75,158 @@ def acceptance_tests(interpreter, browser, options):
     print('Verifying results...')
-    output = os.path.join(env.RESULTS_DIR, 'output.xml')
+    options = []
+    output = os.path.join(RESULTS_DIR, 'output.xml')
     robotstatuschecker.process_output(output, verbose=False)
-    options = [opt.format(browser=browser) for opt in REBOT_OPTIONS]
+    options.extend([opt.format(browser=browser) for opt in REBOT_OPTIONS])
+    options += rf_options
     try:
-        rebot_cli(options + cli_options + [output])
+        rebot_cli(options + [output])
     except SystemExit as exit:
         return exit.code
 
 
+def sauce_credentials(sauce_username, sauce_key):
+    if TRAVIS and not sauce_username and not sauce_key:
+        username = SAUCE_USERNAME
+        key = SAUCE_ACCESS_KEY
+    else:
+        username = sauce_username
+        key = sauce_key
+    return username, key
+
+
 if __name__ == '__main__':
-    if len(sys.argv) < 3 or '--help' in sys.argv:
-        sys.exit(__doc__)
+    parser = argparse.ArgumentParser(
+        description='Script to run Selenium2Library acceptance tests.',
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        epilog=(
+            'Tests are executed using Robot Framework and results verified '
+            'automatically afterwards using `robotstatuschecker` tool. The '
+            'tool can be installed using `pip install robotstatuschecker` '
+            'and more information about it can be found from: '
+            'https://github.com/robotframework/statuschecker/. Notice that '
+            'initially some tests fail.\n\n'
+            'When running test by using browser from Sauce labs, it is '
+            'required that the Sauce Connect is used. The Sauce Connect '
+            'allows the browser from Sauce Labs reach the acceptance test '
+            'web server. The acceptance test uses tunnel with name '
+            '`localtunnel` and therefore when establishing the Sauce Connect '
+            'tunnel use the following command:\n'
+            '`bin/sc -u YOUR_USERNAME -k YOUR_ACCESS_KEY -i localtunnel`\n\n'
+            'More details and to downlaod Sauce Connect visit: '
+            'https://wiki.saucelabs.com/display/DOCS/High+Availability+Sauce+Connect+Setup\n\n'
+            'It is possible to pass Robot Framework command line arguments to '
+            'the test execution as a last arguments to the `run_tests.py`'
+            'command. Example arguments like --test, --suite, --include and '
+            '--exclude may be used to configure acceptance test execution. '
+            'The Robot Framework command line arguments are not displayed in '
+            'the optional arguments list in the help.\n\n'
+            'Examples:\n\n'
+            'python run_tests.py chrome\n'
+            'run_tests.py --interpreter jython firefox --suite javascript\n'
+            'python run_tests.py chrome --sauceusername your_username '
+            '--saucekey account_key --suite javascript\n'
+            ' '
```

Comment:
Should definitely format this using Python's triplequoted strings. Alternatively could use `__doc__` here.